src/components/radioButtonGroupWithTitle.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from "react"
+
+import RadioButton, { RadioButtonProps } from "./radioButton"
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  title: string
+  required?: boolean
+  disabled?: boolean
+  radioInputs: RadioButtonProps[]
+}
+
+function RadioButtonGroup({
```

Comment:
it's totally fine like this but aren't we usually using the arrow function syntax? :D

Reply:
I changed that ! 😃